ompi/communicator/communicator.h

```diff
@@ -261,20 +262,30 @@ struct ompi_communicator_t {
     opal_infosubscriber_t      super;
     opal_mutex_t               c_lock; /* mutex for name and potentially
                                           attributes */
-    char  c_name[MPI_MAX_OBJECT_NAME];
+    char                      *c_name;
```

Comment:
Weird spacing.  Is there some tabs involved here (or in the later lines)?

Reply:
I just tried to align it with the previous and next element in the structure.